From 12a842ee41750225285b9947efc4d9740be50a4f Mon Sep 17 00:00:00 2001
From: gord chung <gord@live.ca>
Date: Thu, 26 Oct 2017 19:46:35 +0000
Subject: [PATCH 48/91] remove branch matching

we don't use it and it might get picked up by other branches

Change-Id: I04da8f73204dffe6081dd412a38510dc34f35901
---
 .zuul.yaml | 20 ++++++--------------
 1 file changed, 6 insertions(+), 14 deletions(-)

diff --git a/.zuul.yaml b/.zuul.yaml
index 7b67ff2..bc05398 100644
--- a/.zuul.yaml
+++ b/.zuul.yaml
@@ -91,14 +91,10 @@
       jobs:
         - ceilometer-dsvm-tempest-plugin-mongodb
         - ceilometer-dsvm-tempest-plugin-mysql
-        - ceilometer-tox-py27-mongodb:
-            branches: ^(?!stable/newton)
-        - ceilometer-tox-py27-mysql:
-            branches: ^(?!stable/newton)
-        - ceilometer-tox-py27-postgresql:
-            branches: ^(?!stable/newton)
+        - ceilometer-tox-py27-mongodb
+        - ceilometer-tox-py27-mysql
+        - ceilometer-tox-py27-postgresql
         - grenade-dsvm-ceilometer:
-            branches: ^(?!stable/newton).*$
             irrelevant-files:
               - ^(test-|)requirements.txt$
               - ^setup.cfg$
@@ -108,14 +104,10 @@
       jobs:
         - ceilometer-dsvm-tempest-plugin-mongodb
         - ceilometer-dsvm-tempest-plugin-mysql
-        - ceilometer-tox-py27-mongodb:
-            branches: ^(?!stable/newton)
-        - ceilometer-tox-py27-mysql:
-            branches: ^(?!stable/newton)
-        - ceilometer-tox-py27-postgresql:
-            branches: ^(?!stable/newton)
+        - ceilometer-tox-py27-mongodb
+        - ceilometer-tox-py27-mysql
+        - ceilometer-tox-py27-postgresql
         - grenade-dsvm-ceilometer:
-            branches: ^(?!stable/newton).*$
             irrelevant-files:
               - ^(test-|)requirements.txt$
               - ^setup.cfg$
-- 
2.7.4

